File: LeetCode/Python/Palindrome_Number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        n = x
        revNum = 0
        if n<0: return False

        while(n>0):
            d = n % 10
            revNum = revNum * 10 + d
            n /= 10
        if revNum == x: return True

File: LeetCode/Java/Palindrome_Number.java
class Solution {
    public boolean isPalindrome(int x) {
        int revNum = 0;
        int n = x;
        if(n<0) return false;

        while(n>0){
            int d = n%10;
            revNum = revNum*10 + d;
            n = n/10;
        }

        if(revNum == x) return true;
        else return false;
    }
}
